package.json

```diff
@@ -107,7 +107,7 @@
     "@types/chai": "^4.0.5",
     "@types/chai-datetime": "^0.0.31",
     "@types/classnames": "^2.2.2",
-    "@types/codemirror": "^0.0.49",
+    "@types/codemirror": "0.0.55",
```

Comment:
Just curious if this PR needed anything from the updated types for codemirror.

Reply:
I couldn't remember but looking at the [commit which introduced that change](https://github.com/desktop/desktop/commit/5bc1c1c0c48ce989849745e06f82a9b2296b9f79) it adds the `execCommand` method, getting rid of an `any` cast.